frontend/src/screens/home/home.tsx

```diff
@@ -13,10 +15,104 @@ interface HomeScreenProps {
 
 const StyledView = styled(View);
 const StyledText = styled(Text);
+const StyledImage = styled(Image);
+
+const WelcomeBlock = ({ name }: { name: string }) => {
+  return (
+    <StyledView className='flex h-[14vh] w-[100vw] items-center justify-center px-[8vw] py-[2vh]'>
+      <StyledView className='flex flex-row w-full h-full'>
+        <StyledView className='flex-1 flex flex-col justify-center space-y-[5vh]'>
+          <StyledText className='text-[2vh] font-nunitoRegular '>Hello {name}!</StyledText>
+          <StyledText className='text-[3vh] font-heading'>Welcome back</StyledText>
+        </StyledView>
+        <StyledView className='flex-2 flex items-center justify-center'>
+          <StyledView className='flex items-center justify-center bg-brand50 w-[12vw] h-[12vw] rounded-full'>
+            <StyledImage
+              source={require('../../../assets/images/notifications.png')}
+              className='w-[7vw] h-[7vw]'
+            ></StyledImage>
+          </StyledView>
+        </StyledView>
+      </StyledView>
+    </StyledView>
+  );
+};
+
+const PortfolioValue = ({
+  Portfoliovalue,
+  Portfoliochange,
+  navigation,
+}: {
+  Portfoliovalue: number;
+  Portfoliochange: number;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Put a comment explaining why you disabled this rule (I'm guessing that the NavigationScreenProp takes in any, any so you didn't have a choice, but good to leave a comment saying that so its documented and future engineers are aware of that).